nss-day-cohort-48/giffygram-looksmart
# Al branch registernewuser

#### Changes Made
1. Modified file `register.js` to include `registration form component` and to send newUser to API.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout al-branch-registernewuser
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > This form should appear when the user clicks on "Register me!", rerendering the site with login form. It then allows user to sign up, checking whether user name or email already exists in database
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
